capacity_factor/info.yml

```diff
@@ -0,0 +1,13 @@
+---
+uid: capacity_factor
+title: Capacity Factor
+prerequisites:
+  - None
```

Comment:
This assessment question depends on knowledge of the definition of availability - should that be a prereq?